draft-ietf-quic-transport.md

```diff
@@ -2527,6 +2546,26 @@ Reordering might cause frames to be received after closing, see
 {{state-hc-remote}}.
 
 
+## Solicited State Transitions
+
+If an endpoint is no longer interested in the data being received, it MAY send a
+STOP_SENDING frame on a stream in the "open" or "half-closed (local)" state to
+request closure of the stream in the opposite direction.  This typically
+indicates that the receiving application is no longer reading from the stream,
+but is not a guarantee that incoming data will be ignored.
+
+STREAM frames received after sending STOP_SENDING are still counted toward the
+connection and stream flow-control windows.  Even though these frames will be
+discarded, because they are sent before their sender receives the STOP_SENDING,
+the sender will consider the frames to count against its flow-control windows.
+
+Upon receipt of a STOP_SENDING frame on a stream in the "open" state, an
+endpoint SHOULD send a RST_STREAM with an error code of QUIC_RECEIVED_RST.  If
+the STOP_SENDING frame is received on a stream that is already in the
+"half-closed (local)" or "closed" states, a RST_STREAM frame MAY still be sent
+in order to cancel retransmission of previously-sent STREAM frames.
```

Comment:
This wording seems a bit strange to me. Why does sending a RST_STREAM frame cancel retransmission of STREAM frames, isn't canceling retransmission a decision made by the sender locally?